### PR TITLE
Ref/push notification chainservice

### PR DESCRIPTION
**Pull request by matiu**

### CODE REVIEW

File: packages/bitcore-wallet-service/src/lib/blockchainexplorers/v8.ts [hunk 1]
```diff
@@ -489,7 +487,7 @@ export class V8 {
 
     blockSocket.on('connect', () => {
       log.info(`Connected to block ${this.getConnectionInfo()}`);
-      blockSocket.emit('room', `/${this.coin.toUpperCase()}/${this.v8network}/inv`);
+      blockSocket.emit('room', `/${this.chain}/${this.v8network}/inv`);
```
Comment (micahriggan) [inline packages/bitcore-wallet-service/src/lib/blockchainexplorers/v8.ts:490]: this.chain is lowercased up above right? I don't think that'd be a valid room for bitcore-node

File: packages/bitcore-wallet-service/src/lib/blockchainexplorers/v8.ts [hunk 2]
```diff
@@ -122,7 +120,7 @@ export class V8 {
       name: wallet.id,
       pubKey: wallet.beAuthPublicKey2,
       network: this.v8network,
-      chain: this.coin
+      chain: this.coin // lowerCase?
```
Comment (micahriggan) [inline packages/bitcore-wallet-service/src/lib/blockchainexplorers/v8.ts:123]: I think this should be this.chain

Comment (matiu) [reply packages/bitcore-wallet-service/src/lib/blockchainexplorers/v8.ts:123]: I agree. but it seems we are sending it lowercase now, so I didnt wanted to modify it.

Is it possible we have something on bitcore-node what allows chain to be lowerCased on wallet registration?

Comment (micahriggan) [reply packages/bitcore-wallet-service/src/lib/blockchainexplorers/v8.ts:123]:
```
  let { chain, network } = req.params;
  let { name, pubKey, path, singleAddress } = req.body;
```

Looks like it's coming from the URL, and not the request body

Comment (matiu) [reply packages/bitcore-wallet-service/src/lib/blockchainexplorers/v8.ts:123]: ic, ok, ill change it anyways so it is clear.

**Pull request merged**